openupgrade_scripts/scripts/uom/14.0.1.0/upgrade_analysis.txt

```diff
@@ -3,6 +3,7 @@
 uom          / uom.category             / measure_type (selection)      : DEL selection_keys: ['length', 'unit', 'volume', 'weight', 'working_time']
 uom          / uom.uom                  / measure_type (selection)      : DEL selection_keys: function
 ---XML records in module 'uom'---
+DEL ir.model.constraint: uom.constraint_uom_category_uom_category_unique_type
```

Comment:
We must remove this constraint manually in the `uom` migration scripts.

Reply:
Done